app/javascript/pages/admin/Events/queries/index.gql

```diff
@@ -1,12 +1,16 @@
 #import "fragments/EventEntry.gql"
 #import "fragments/OfficeEntry.gql"
+#import "fragments/UserEntry.gql"
 
 query EventsQuery {
   events {
     ...EventEntry
     office {
       ...OfficeEntry
     }
+    users {
```

Comment:
I'm not crazy about doing this on the event collection... thats a lot of data to shove into the response. Where is this used?

Reply:
Oh you're right this was a mistake. Will remove it